Lib/test/test_zoneinfo/__init__.py

```diff
@@ -0,0 +1 @@
+from .test_zoneinfo import *
```

Comment:
Why not putting test content in this file, as done by test_warnings/__init__.py for example, rather than having test_zoneinfo/test_zoneinfo.py?

Reply:
There's a few reasons for this. The most immediate reason is that in the reference implementation there's also a `test_zoneinfo_property.py`, which has all the property tests for this module. Once feature freeze is done, I'm planning to make the case for adding property tests to the standard library, at which point we'll want the parallel structure anyway.

Even if that doesn't happen, I'm considering a future modification to the test similar to the way the datetime tests are split between `datetimetester.py` and `test_datetime.py`, and `test_datetime.py` generates C and pure Python tests from just the C tests (as opposed to the way I'm manually doing it now).

Reply:
Oh ok. I was more a question than a request. I'm fine with the current file layout.